src/electionguard/ballot.py

```diff
@@ -816,9 +816,9 @@ class BallotBoxState(Enum):
 
 
 @dataclass(unsafe_hash=True)
-class CiphertextAcceptedBallot(CiphertextBallot):
+class SubmittedBallot(CiphertextBallot):
     """
-    A `CiphertextAcceptedBallot` represents a ballot that is accepted for inclusion in election results.
+    A `SubmittedBallot` represents a ballot that is accepted for inclusion in election results.
     An accepted ballot is or is about to be either cast or spoiled.
```

Comment:
Hi @OBITORASU , "An accepted ballot" and similar occurrences should be changed to "A submitted ballot". Please update your PR :) Thanks

Reply:
On it!

Reply:
Just a small doubt. Do I cover all the instances of ```accepted_ballot``` as well?

Reply:
I think so, @keithrfung , please second me.

Reply:
My commit is pretty much ready and waiting once I get the final response I will do the needful and push :)

Reply:
Cool, thanks

Reply:
thanks for jumping in folks! we're following up about your question and will get back pronto. 👍

Reply:
Sorry i'm late on the response. 

Yes. `Accepted` as a term is getting changed to `Submitted` including comments and functions.

Reply:
Also don't forget to check the linting. `make auto-lint`

Reply:
Thanks for the clarification! I will push the commit soon.